scripts/notify.sh

```diff
@@ -69,7 +69,7 @@ if [[ ! -f "$PID_FILE_PATH" ]]; then  # If pane not yet monitored
         tmux select-window -t @"$WINDOW_ID"
         tmux select-pane -t %"$PANE_ID"
       fi
-      notify "$complete_message" "$complete_title" $2
+      SESSION_ID=$SESSION_ID WINDOW_ID=$WINDOW_ID notify "$complete_message" "$complete_title" $2
```

Comment:
```suggestion
      notify "$complete_message" "$complete_title" $2
```

I don't think you need to map the `SESSION_ID` and `WINDOW_ID` parameters again since notify is executed in the same shell where these variables are already sourced.